doc/common/abbreviations.rst

```diff
@@ -98,6 +99,7 @@ DSP            Digital Signal Processor
 DU             Distributed Unit (O-RAN context)
 DVR            Distributed Virtual Routing
 E2E            End to End
+EBPF           Extended Berkley Packet Filter
```

Comment:
Usually this is written with a small "e" like eBPF. I'm not sure if this should be followed here...

Reply:
Yes, as per the PR description.  My fault, corrected.